app/tests/responses/test_endpoint_responses.py

```diff
@@ -35,12 +35,12 @@ def test_get_allowed_origin_should_return_wildcard_when_env_is_dev(self, mock_co
     @patch('hallebarde.responses.endpoint_responses.config')
     def test_get_allowed_origin_should_return_website_hostname_when_env_is_not_dev(self, mock_config):
         # Given
-        website_hostname = 'https://ourwebsite'
+        website_hostname = 'ourwebsite'
         mock_config.ENVIRONMENT = 'prod'
         mock_config.WEBSITE_HOSTNAME = website_hostname
 
         # When
         origin = endpoint_responses.get_allowed_origin()
 
         # Then
-        assert origin == website_hostname
+        assert origin == 'https://'+website_hostname
```

Comment:
Does this test pass if we execute it locally ? (I have some doubt about it with the `https` for local tests)

Reply:
Actually it does, as this just tests if the function returns the correct string and not the actual HTTPS connection